viam-cartographer/src/carto_facade/carto_facade_test.cc

```diff
@@ -1,5 +1,9 @@
 #include "carto_facade.h"
 
+#include <pcl/conversions.h>  //pcl::fromPCLPointCloud2
```

Comment:
[nit] //pcl -> // pcl

Also, should we say "for" in all of these, e.g. "for pcl::fromPCLPointCloud2"?

Reply:
done